fastchat/conversation.py

```diff
@@ -163,6 +164,14 @@ def get_prompt(self) -> str:
                 else:
                     ret += role + "\n"
             return ret
+        elif self.sep_style == SeparatorStyle.CHATGLM3:
```

Comment:
Can you add reference?

Reply:
How about adding this ref (https://huggingface.co/THUDM/chatglm3-6b/blob/fc3235f807ef5527af598c05f04f2ffd17f48bab/tokenization_chatglm.py#L184) ?

Reply:
@ZeyuTeng96 Like this [conversation.py#L167](https://github.com/lm-sys/FastChat/blob/main/fastchat/conversation.py#L167) :)